GVFS/GVFS.Virtualization/Projection/GitIndexProjection.GitIndexEntry.cs

```diff
@@ -18,11 +21,26 @@ internal class GitIndexEntry
             private const int MaxParts = MaxPathBufferSize / 2;
             private const byte PathSeparatorCode = 0x2F;
 
+            private static readonly string PathSeparatorString = Path.DirectorySeparatorChar.ToString();

```

Comment:
It looks like you want the platform-dependent path separator here - should this live on `GVFSPlatform` instead? I think that'll make the intent more clear.